dts/common/yaml/sensor.yaml

```diff
@@ -0,0 +1,30 @@
+---
+title: Sensor Base Structure
```

Comment:
this could probably be more generically just an i2c device.  Nothing sensor specific

Reply:
so i2c-device.yaml?

Reply:
yes